.kmip/kmip_server/set-temp-creds.sh

```diff
@@ -0,0 +1,40 @@
+# Obtains temporary AWS credentials for CSFLE testing.
```

Comment:
I think `set-temp-creds.sh` can be removed since it is not part of the instructions.